modules/Administration/PasswordManager.php

```diff
@@ -81,7 +81,8 @@ function clearPasswordSettings() {
 $valid_public_key= true;
 if(!empty($_POST['saveConfig'])){
     if ($_POST['captcha_on'] == '1'){
-		$handle = @fopen("http://www.google.com/recaptcha/api/challenge?k=".$_POST['captcha_public_key']."&cachestop=35235354", "r");
+		(isset($_SERVER['HTTPS']) && 'on' === $_SERVER['HTTPS']) ? $protocol = 'https' : $protocol = 'http';
```

Comment:
Apparently this won't work with IIS maybe you can change it to something like this
http://stackoverflow.com/a/2886224

`if ((isset($_SERVER['HTTPS']) && 'on' === $_SERVER['HTTPS']) ||
            (isset($_SERVER['SERVER_PORT']) && 443 === $_SERVER['SERVER_PORT'])
        ) {
            $protocol = 'https';
        } else {
            $protocol = 'http';
        }`